apps/test/unit/code-studio/components/progress/ScriptOverviewTopRowTest.jsx

```diff
@@ -166,6 +170,65 @@ describe('ScriptOverviewTopRow', () => {
     ).to.be.true;
   });
 
+  it('renders the unit calendar when showCalendar true for teacher', () => {
+    const wrapper = shallow(
+      <ScriptOverviewTopRow
+        {...defaultProps}
+        showCalendar
+        lessons={testLessons}
```

Comment:
FYI: looks like the props in the test file need to be update to match the updates you made to the component